### DISCUSSION TITLE
小米无线开关只有电量，没有开关动作

### DISCUSSION THREAD

**Discussion by boysa23000000000** [XiaoMi/ha_xiaomi_home#374] [Ideas]
小米无线开关在HA只有电量，没有开关动作。无法添加自动化。

**Comment by SusanPhevos** [comment]
请问设备详情页的 model 是什么？
<img width="348" height="267" alt="Screenshot from 2025-11-26 17-32-24" src="https://github.com/user-attachments/assets/d6aac6f9-d6b8-432d-b3af-0c49fd8b477e" />
